plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaClientAdminFactory.java

```diff
@@ -18,33 +18,33 @@
 
 import javax.inject.Inject;
 
+import java.util.Map;
 import java.util.Properties;
-import java.util.Set;
 
 import static java.util.Objects.requireNonNull;
 import static java.util.stream.Collectors.joining;
 import static org.apache.kafka.clients.consumer.ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG;
 
-public class PlainTextKafkaAdminFactory
+public class KafkaClientAdminFactory
```

Comment:
Why not to have separate `SslKafkaAdminFactory` to be used only when `kafka.security-protocol=SSL`?

Also we could inject `PlainTextKafkaAdminFactory` as a delegate to be used here, like:
```
    @Override
    public Properties configure()
    {
        // delegate is PlainTextKafkaAdminFactory
        Properties properties = delegate.configure();
        properties.putAll(sslConfiguration);
        return properties;
     }
```

Reply:
Hello @kokosing ,

Yes, this should be fixed after I merge my 2nd commit. `KafkaClientsModule.java` will set the configs based on `SSL` or `PLAINTEXT`.